packages/components/form/src/form-item.vue

```diff
@@ -1,19 +1,26 @@
 <template>
-  <div ref="formItemRef" :class="formItemClasses">
+  <div
+    ref="formItemRef"
+    :class="formItemClasses"
+    :role="isGroup ? 'group' : undefined"
+    :aria-labelledby="isGroup ? labelId : undefined"
+  >
     <form-label-wrap
       :is-auto-width="labelStyle.width === 'auto'"
       :update-all="formContext?.labelWidth === 'auto'"
     >
-      <label
-        v-if="label || $slots.label"
+      <component
+        :is="labelFor ? 'label' : 'div'"
+        v-if="hasLabel"
+        :id="labelId"
         :for="labelFor"
```

Comment:
**Question:** would this `:for` attribute be redundant when the tag is actually `div` instead of `label`?

Reply:
Yes, in this case the "labelFor" computed returns undefined which removes the attribute.